### PR TITLE
Job edits

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: _episodes/12-scheduler.md [hunk 1]
```diff
@@ -165,12 +170,18 @@ which is probably not what we want.
 
 The following are several key resource requests:
 
+* `-n <nnodes>` - how many nodes does your job need?
```
Comment (jstaf) [inline _episodes/12-scheduler.md]: I'll admit I didn't want to get too far into multi-node jobs for these intro materials... that said, `-n` specifies number of tasks which does not necessarily correspond to number of nodes if `-c` is set (cpus per task). Could you add a bit of clarification here on this and then I'll merge this pull request?

Comment (ChristinaLK) [reply _episodes/12-scheduler.md]: I was lazy and didn't look up the right option for nodes ( `-N`).  Oops. 

I was thinking bigger picture -- having a list of the common options that people should know about (whether or not they need them immediately).

Comment (jstaf) [reply _episodes/12-scheduler.md]: Ok sounds good - I will add some clarification and merge this in (I might move the an overview of the differences between `-n` and `-N` into a callout box somewhere).

**Comment by jstaf** [comment]
In light of the potential upcoming changes to this repository, I'm going to merge this for now and add my original `-n`/`-N` distinction as an issue on the repository. (trying to get the remaining pull requests merged)

**Pull request merged**